src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -427,6 +427,25 @@
     <RemoveDir Directories="$(OutputPath)/15.0" />
   </Target>
 
+  <!-- Patch version of Newtonsoft.Json in testhost.deps.json, because it is picked up by 
+  compliance checking scripts as a false-positive vulnerability. -->
+  <Target Name="FixNewtonsoftInTesthostDepsJson"
+          BeforeTargets="PublishStage2WithBackwardsCompatibleRuntimes"
+          AfterTargets="Build">
+    <PropertyGroup>
+      <ReplacementPattern>13.0.0.0"</ReplacementPattern>
```

Comment:
Question, this will update the assembly version as well and from what I can tell, that version is still 13.0.0.0 in other deps.json files. Is that a problem and should we maintain the assembly version?

Reply:
Addressed in the new PR.